juju/client/_client1.py

```diff
@@ -43,7 +43,7 @@ class ActionPrunerFacade(Type):
     
 
     @ReturnMapping(ModelConfigResult)
-    async def ModelConfig(self):
```

Comment:
kwargs in responses makes a little bit of sense, but kwargs in requests feels a bit wrong. Either you know what to send or you don't. You don't happen to know just what to put in kwargs. If you *did* then you should be using a newer client lib that supports it, shouldn't you?
(kwargs in responses is more about not causing a runtime error because extra data was returned, but by the same token, it doesn't really matter because you can't know what to do with things that you don't know about yet.)
I guess all of these ignore anything passed in **kwargs, so it is just dropped?

Reply:
I've addressed this in the later commits, by dropping kwargs on requests and also pinning to a facade version. Moving to a later facade requires understanding the changes and things don't come for free and requires thought and coding to move to a later facade.